nixos/modules/installer/cd-dvd/sd-image.nix

```diff
@@ -146,11 +146,14 @@ in
           echo "file sd-image $img" >> $out/nix-support/hydra-build-products
         fi
 
+        root_img="./root-fs.img"
```

Comment:
No real need to make this a variable, as it's basically static and self-contained in the `sd-image.nix` build...

But, no real need to change it either. Just saying.